tools/docker/gen_dockerfile.py

```diff
@@ -0,0 +1,58 @@
+# SPDX-License-Identifier: MPL-2.0
+
+import argparse
+import os
+import sys
+import logging
+from jinja2 import Environment, FileSystemLoader
```

Comment:
Just to be sure. Has jinja2 installed by default on Ubuntu? If not, the Dockerfile needs to be updated to include the installation instruction for jinja2.

Reply:
No, but do we need to build the Dockerfile in the Asterinas docker container? I think it should be built in the host environment.

Reply:
I see. You are right.